### ISSUE TITLE
RFFT support in tensorflow

### ISSUE THREAD

**Issue by cvasfi** [el3ment/spectral_representations_for_convolutional_neural_networks#4]
Since RFFT is supported in tensorflow now, which parts of the code would you suggest to tweak? 

I think the big loop in the spectral_to_variable would be one of them. what about the fftshift functions, are they also unnecessary?

**Comment by el3ment** [comment]
Yes! It's been awhile since I've looked at this code -- but I think you are correct.
